tools/test.py

```diff
@@ -697,6 +697,7 @@ def Execute(args, context, timeout=None, env={}, faketty=False):
     fd_in = fd_err = fd_out
     pty_out = out_master
   else:
+    fd_in = None
```

Comment:
I guess someone made a similar fix to me and I didn't notice when I rebased. I'm not very accomplished with Git. Fixed.